src/imitation/util/util.py

```diff
@@ -25,6 +25,34 @@
 from stable_baselines3.common.vec_env import DummyVecEnv, SubprocVecEnv, VecEnv
 
 
+def oric(x: np.ndarray) -> np.ndarray:
+    """Optimal rounding under integer constraints.
+
+    Given a vector of real numbers such that the sum is an integer, returns a
+    vector of rounded integers that preserves the sum and which minimizes the
+    Lp-norm of the difference between the rounded and original vectors. Algorithm
```

Comment:
p isn't a parameter of the function, so which Lp norm have we chosen to minimize? (I think the L1?)

Reply:
I think L1 is what we care about, but the method actually guarantees that _all_ Lp norms with p >= 1 are minimized by the resulting rounded values (see Proposition 3 in the paper).

Reply:
thanks, clarified in docstring